src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,419 @@
+.pf-c-wizard {
+  --pf-c-wizard--GridTemplateRows: auto auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateRows: auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateColumns: 300px 1fr;
+  --pf-c-wizard--Width: calc(100% - var(--pf-global--spacer--xl) * 2);
```

Comment:
Does it need a width and height?

Reply:
It's similar to a modal, so I think it would. That said, I'll verify with Kyle that it will always present as a modal, and assuming it fits in a `.pf-l-bullseye` I'll make sure the height/width works the same as a modal

Reply:
@kybaker said to match about modal box sizing. updated to match.